website/docs/guides/howto-guide.html.markdown

```diff
@@ -0,0 +1,107 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas Provider How to Guide"
+sidebar_current: "docs-mongodbatlas-guides-how-to-guide"
+description: |-
+MongoDB Atlas Provider : How to Guide
+---
+
+# MongoDB Atlas Provider: How to Guide
+
+The Terraform MongoDB Atlas Provider guide to perform common tasks with the provider.
+
+Document for users how to get a pre-existing container id
+
+1) Empty project
+
+2) empty state file
+
+3) applied curl to build cluster
+
+4) terraform apply to retrieve container id
+
+Validated by testing following scenario container_id populated in state after first apply ...
+
+1) create cluster using curl command to simulate non terraform based cluster
+
+```
+curl --user "pub:priv" --digest \
+--header "Content-Type: application/json" \
+--include \
+--request POST "https://cloud.mongodb.com/api/atlas/v1.0/groups/grpid/clusters?pretty=true" \
+--data '
+{
+  "name": "SingleRegionCluster",
+  "diskSizeGB": 100,
```

Comment:
Trying to get users to get out of the habit of using this one unless the want more than the default storage so let's remove it as it should be optional.  Though we should test that as the docs makes it sound like it is required but it shouldn't be. 
```suggestion
```